src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -132,43 +132,32 @@ public boolean hasRentalInformation(Client client, RentalInformation rentalInfor
         return addressBook.hasRentalInformation(client, rentalInformation);
     }
 
-    //=========== Filtered Client List Accessors =============================================================
+    //=========== Filtered and Sorted Client List Accessors ==========================================================
 
     /**
      * Returns an unmodifiable view of the list of {@code Client} backed by the internal list of
      * {@code versionedAddressBook}
      */
     @Override
     public ObservableList<Client> getFilteredPersonList() {
-        return filteredClients;
+        return sortedFilteredClients;
     }
 
     @Override
     public void updateFilteredPersonList(Predicate<Client> predicate) {
         requireNonNull(predicate);
         filteredClients.setPredicate(predicate);
 
-        if (filteredClients.isEmpty()) {
+        if (sortedFilteredClients.isEmpty()) {
```

Comment:
same apply here, is there a specific reason to check if sortedFilteredClietns is empty instead of filteredClients?

Reply:
Same as above, because the list we are seeing is both sortable and filterable